news/2024/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks.md

```diff
@@ -0,0 +1,291 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Following up on the previous question, what are your thoughts on the current state of completionist badge?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/4568537" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-Predominador.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/4568537">Predominador</a></p>
+
+I think that the badges are really cool, it needs to be updated more frequently, though... but PERSONALLY, I feel titles are more cool in completionist cases. It's very very unique thing to do, and it just keeps getting harder and harder to complete the game, since there are maps getting ranked everyday!
+But related to that, same as what to happens in [this page](/wiki/People/Completionists), I feel it would be cool to have some page in the wiki that register players that grind other things and reach specific marks too, and when. As for example SS and Ranked Score, which are some of other things that many usually grind nowadays. Who already reached 50k SS in osu!catch and WHEN did they reach it? and in osu!? Taiko? osu!mania? I feel those things are important not only to keep history registered, but to all players that grinds other things also get recognized by their efforts too, which is usually understimated and underappreciated.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+As an osu!catch player, what are your thoughts on Lazer?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/4568537" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-Predominador.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/4568537">Predominador</a></p>
+
+I didn't play that much osu!catch in Lazer, but I used to play in Lazer to play Spotlights, and I saw it getting better and better over time. It feels good to play, even though I suffer some spike lags if I don't close everything else. But this is also probably because my PC is very very old ;c
```

Comment:
```suggestion
I didn't play that much osu!catch in osu! (lazer), but I used to play Spotlights in osu!(lazer), and I saw it getting better and better over time. It feels good to play, even though I suffer some spike lags if I don't close everything else. But this is also probably because my PC is very very old ;c
```

Reply:
Probably this sounds nice?